docusaurus/docs/infrastructure/testing/load_test_1.md

```diff
@@ -0,0 +1,259 @@
+| GatewayCount            | 1              | 10             | +1 gateway every 100 blocks                   |
+| ApplicationCount        | 5              | 1,000          | +10 apps every 10 blocks                      |
+| SupplierCount           | 5              | 1,00           | +1 every 100 blocks                           |
+| ProxyService / DataNode | 0 / ∞          | 0 / ∞          | Mocked to avoid being a performance bottlneck |
+
+### Constant Parameter
+
+- `BlockTime` - A constant block time between 10s and 60s will be selected for the benchmarks in this test
+- `RequestType` - We will use a “dummy” backing data node / proxy service that leverages `nginx` to return a `200` or 500 randomly
+- `RequestDistribution` - The `RelaysPerSecond` will be evenely distributed
+- `VirtualUsers` - For simplicity, we will assume a `1:1` mapping of virtual users (i.e. curl clients) to `Applications`
+
+### Out-of-scope Parameters
+
+- `Governance Parameters` have not been implemented yet and are therefore out-of-scope
+
+## Measurements
+
+### What to measure?
+
+#### 1. Chain State Size
+
+`What`:
+
+- A `pie chart` or `stacked bar chart` of how the data in the Blocks is distributed
+- A `line chart` showing state growth over size
+
+`Why`
+
+- Get an estimate of the cost of data publishing (i.e. TIA tokens)
+- Get an estimate of data distribution (where to focus short-term optimizaiton efforts)
+
+`Example:`
+
+```mermaid
+pie showData
+    title State Distribution in one block
+    "Proofs" : 30
+    "Claims" : 20
+    "Applications" : 10
+    "Suppliers" : 10
+    "Gateways" : 10
+    "Signatures" : 10
+    "Misc" : 10
+```
+
+#### 2. Sequencer (Validator, etc…)
+
+`What`: Multiple `line charts` to capture `Disk` (size & iops), `RAM`, `CPU` , `Network` usage (ingress/egress)
+
+`Why`:
+
+- `Proof Validation` - `RAM` & `CPU` could be a potential bottleneck
+- `Block generation` - `RAM` & `CPU` could be a bottleneck in preparing new blocks
+- `Block Publishing` - `Tx` aggregation (ingress) and `Block` publishing (egress) could be more expensive than expected w.r.t network usage
+- `Rollup State` - `Disk` could be a limiting factor depending on how quickly state grows
+
+|                  | RAM  | CPU  | Network | Disk |
+| ---------------- | ---- | ---- | ------- | ---- |
+| Proof Validation |   ❓ |   ❓ |         |      |
+| Block Generation |   ❓ |   ❓ |         |      |
+| Block Publishing |      |      |   ❓    |      |
+| Rollup State     |      |      |         |   ❓ |
+
+#### 3. AppGate Server (Application, Gateway, etc…)
+
+`What`: Multiple `line charts` to capture `Disk` (size & iops), `RAM`, `CPU` , `Network` usage (ingress/egress)
+
+`Why`:
+
+- `Relay Proxies` Ingress/egress of relays could add
+- `Caches & State` - All the caching & state can have impact across the board
+- `Request Processing` - Signature generation, request marshaling / unmarshaling, etc…
```

Comment:
Slow `Supplier` replies could increase the pending relays at the `AppGate` level (RAM)